conda-env/pub.yml

```diff
@@ -13,6 +13,18 @@ dependencies:
   # ==================
   - python=3.8.10
   - pip=22.0.3
+  - pyyaml=6.0
+  - cmor=3.6.1
+  - netcdf4=1.5.6
```

Comment:
`netcdf4=1.5.8` (the latest version) conflicts with `autocurator=0.1`.

Have to fall back to 1.5.6.

Reply:
Ok by me.  But Sasha should be pulled on this - not to approve, but just to make aware.